templates/composer.json.template

```diff
@@ -8,7 +8,7 @@
     "homepage": "https://grpc.io",
     "license": "Apache-2.0",
     "require": {
-      "php": ">=5.5.0"
+      "php": ">=7.0.0"
```

Comment:
So you will need to run the `tools/buildgen/generate_projects.sh` script to make sure these changes in the `templates/` directory are reflected in the _real_ file, in this case, the `composer.json` file at the root directory. Both changes (the file under the `templates/` directory and the real file) need to be checked in together.

Reply:
thanks